core/src/mindustry/content/Blocks.java

```diff
@@ -1589,6 +1589,26 @@ public void drawBase(Tile tile){}
             reloadTime = 10f;
         }};
 
+        tsunami = new LiquidTurret("tsunami"){{
+            requirements(Category.turret, with(Items.metaglass, 100, Items.lead, 400, Items.titanium, 250, Items.thorium, 100));
+            ammo(
+                    Liquids.water, Bullets.waterShot,
```

Comment:
Should use different bullet variants that deal more damage, draw larger and have more knockback.